### ISSUE TITLE
Church Encoding

### ISSUE THREAD

**Issue by lesnitsky** [lesnitsky/11001101110000#1]
zero f x = x
one f x = f x
two f x = ... one ... ??? WTF

inc n = ... (n + 1) ... WTF?!?!

**Comment by DaQuirm** [comment]
That's right @Drapegnik, another developer is discovering the joy of lambda calculus 😄
